openshift/setup-teardown/params/CUSTOM_SETTINGS/EAGLE_OS4/admin/dev/admin-4-angular-on-nginx-dc.params

```diff
@@ -6,3 +6,7 @@ TAG_NAME=dev
 BANNER_COLOUR=orange
 REMOTE_API_PATH=https://eagle-dev.apps.silver.devops.gov.bc.ca/api
 REMOTE_PUBLIC_PATH=https://eagle-dev.apps.silver.devops.gov.bc.ca/
+NODEJS_CPU_REQUEST=10m
```

Comment:
our clamav  pod wasn't able to boot because  it has these requests and limits, are these standard values or something? We  might want to make these a bit higher?

Reply:
This is for the admin nginx pod.  It uses very little resources

Reply:
I see, alright just wanted to make sure we didn't run into the same issue

Reply:
I will update these param names to `NGINX_` to be more correct

Reply:
oh okay, good idea